--- alsa-lib-1.0.5/include/alsa-symbols.h.orig	2003-08-13 13:18:36.000000000 +0000
+++ alsa-lib-1.0.5/include/alsa-symbols.h	2004-05-31 19:14:48.000000000 +0000
@@ -56,6 +56,11 @@
 	__asm__ (".weak ." #name); 			\
 	__asm__ (".set " #name "," #real);		\
 	__asm__ (".set ." #name ",." #real)
+#elif defined(__alpha__)
+#define use_default_symbol_version(real, name, version) \
+	__asm__ (".weak " #name);			\
+	__asm__ ("$" #name "..ng = $" #real "..ng");	\
+	__asm__ (#name " = " #real);
 #else
 #define use_default_symbol_version(real, name, version) \
 	__asm__ (".weak " #name); \
