--- alsa-lib-1.0.8/src/pcm/pcm_dmix.c.orig	2005-01-11 18:42:49.000000000 +0100
+++ alsa-lib-1.0.8/src/pcm/pcm_dmix.c	2005-05-03 14:47:10.000000000 +0200
@@ -53,12 +53,13 @@
  *
  */
 
+static int shm_sum_discard(snd_pcm_direct_t *dmix);
+
 /*
  *  sum ring buffer shared memory area 
  */
 static int shm_sum_create_or_connect(snd_pcm_direct_t *dmix)
 {
-	static int shm_sum_discard(snd_pcm_direct_t *dmix);
 	struct shmid_ds buf;
 	int tmpid, err;
 	size_t size;
--- alsa-lib-1.0.8/src/conf.c.orig	2004-10-05 17:33:04.000000000 +0200
+++ alsa-lib-1.0.8/src/conf.c	2005-05-03 14:53:20.000000000 +0200
@@ -2494,9 +2494,11 @@
  *               specified by \p result.
  * \return Zero if successful, otherwise a negative error code.
  */
+
+static int snd_config_hooks(snd_config_t *config, snd_config_t *private_data);
+
 int snd_config_search_hooks(snd_config_t *config, const char *key, snd_config_t **result)
 {
-	static int snd_config_hooks(snd_config_t *config, snd_config_t *private_data);
 	SND_CONFIG_SEARCH(config, key, result, \
 					err = snd_config_hooks(config, NULL); \
 					if (err < 0) \
@@ -2516,7 +2518,6 @@
  */
 int snd_config_searcha_hooks(snd_config_t *root, snd_config_t *config, const char *key, snd_config_t **result)
 {
-	static int snd_config_hooks(snd_config_t *config, snd_config_t *private_data);
 	SND_CONFIG_SEARCHA(root, config, key, result,
 					snd_config_searcha_hooks,
 					err = snd_config_hooks(config, NULL); \
