--- alsa-lib-1.0.0rc2/utils/alsa.m4.orig	Mon May 27 13:14:20 2002
+++ alsa-lib-1.0.0rc2/utils/alsa.m4	Fri Dec 19 12:10:43 2003
@@ -12,7 +12,7 @@
 dnl For backwards compatibility, if ACTION_IF_NOT_FOUND is not specified,
 dnl and the alsa libraries are not found, a fatal AC_MSG_ERROR() will result.
 dnl
-AC_DEFUN(AM_PATH_ALSA,
+AC_DEFUN([AM_PATH_ALSA],
 [dnl Save the original CFLAGS, LDFLAGS, and LIBS
 alsa_save_CFLAGS="$CFLAGS"
 alsa_save_LDFLAGS="$LDFLAGS"
