--- alsa-lib-1.1.9/configure.ac~	2019-05-10 08:57:21.000000000 +0200
+++ alsa-lib-1.1.9/configure.ac	2019-11-02 13:31:09.773236167 +0100
@@ -423,7 +423,7 @@
   pythonlibs0=
   pythoninc0=
   if test "$build_python2" != "yes"; then
-    pythonlibs0=$(python3-config --libs)
+    pythonlibs0=$(python3-config --libs --embed)
     pythoninc0=$(python3-config --includes)
   fi
   if test -z "$pythonlibs0"; then
